agent/engine/engine_unix_integ_test.go

```diff
@@ -172,6 +172,7 @@ func createVolumeTask(scope, arn, volume string, autoprovision bool) (*apitask.T
 		DriverOpts: map[string]string{
 			"device": tmpDirectory,
 			"o":      "bind",
+			"type":   "tmpfs",
```

Comment:
this was added for compatibility with newer docker versions, see https://github.com/docker/engine/commit/d5b271c155926057d33c2128ff57e49274455813